source/documentation/monitoring_apps/metrics.md

```diff
@@ -171,21 +171,21 @@ Redis is an open source in-memory data store.
 
 #### CPU utilisation
 
-How much computational work your Redis service instance is doing. If you think your CPU utilisation is too high, you can optimise your database queries or upgrade your service plan.
+How much computational work your Redis service instance is doing. If you think your CPU utilisation is too high, you can optimise how you read data from and write data to Redis, or upgrade your service plan.
 
 #### Memory used
 
-How much memory your Redis service instance is using to run itself and to store your app data. Memory limit is pre-set based on service plan and you cannot change this memory limit. If your service instance reaches its memory limit, it will start [evicting data keys](monihttps://redis.io/topics/lru-cache). This may indicate that you need to review the size of the data you're storing, or that you need to upgrade your service plan.
+How much memory your Redis service instance is using to run itself and to store your app data. Memory limit is pre-set based on service plan and you cannot change this memory limit. If your service instance reaches its memory limit, it will start [evicting data keys](https://redis.io/topics/lru-cache). This may indicate that you need to review the size of the data you're storing, or that you need to upgrade your service plan.
 
 #### Swap memory used
 
-If your Redis service instance is running low on memory, it will start to swap memory onto the hard disk. To reduce memory swapping, you can reduce the number of apps using your service instance or upgrade your service plan.
+If your Redis service instance is running low on memory, it will start to swap memory onto the hard disk. To reduce memory swapping, you can reduce the amount of memory your apps use or upgrade your service plan.
```

Comment:
Apologies, looking at my previous comment I can see that it was ambiguous. I meant the amount of data being used by Redis.

Reply:
do we need to tell them how to do this?

Reply:
I don't think so. We explained that in the bit about memory consumption iirc

Reply:
ok